### DISCUSSION TITLE
Support for Microsoft Teams Mentions

### DISCUSSION THREAD

**Discussion by HeedfulCrayon** [cake-build/cake#4169] [Extension requests]
Microsoft Teams has released support for allowing mentions via webhook.  This would be beneficial to implement so that you could get notifications about your own build.
Information about mentions via webhooks can be found here https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-format?tabs=adaptive-md%2Cdesktop%2Cconnector-html#user-mention-in-incoming-webhook-with-adaptive-cards

**Comment by devlead** [comment]
This is probably a good addition to the Microsoft Teams cake addin

https://github.com/cake-contrib/Cake.MicrosoftTeams
